docs/Authoring.md

```diff
@@ -125,6 +125,23 @@ directory by using `Link` metadata to specify file path in output package.
 </ItemGroup>
 ```
 
+To package content v2 files, use `ContentFile` item type. This item type supports 
+`PackageBuildAction`, `PackageCopyToOutput` and `PackageFlatten` metadata. 
```

Comment:
On one hand, I wanted to preserve NuGet attribute naming, on other hand `CopyToOutput` was in conflict with regular MSBuild property for items. So I decided to add `Package` prefix to the NuGet attribute names and call it a day.